plugins/orchestrator/src/components/WorkflowDefinitionViewerPage/WorkflowDefinitionDetailsCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { InfoCard } from '@backstage/core-components';
+import { AboutField } from '@backstage/plugin-catalog';
+
+import { Grid } from '@material-ui/core';
+
+import { WorkflowOverview } from '@janus-idp/backstage-plugin-orchestrator-common';
+
+import WorkflowOverviewFormatter from '../../dataFormatters/WorkflowOverviewFormatter';
+
+const WorkflowDefinitionDetailsCard = ({
+  workflowOverview,
+}: {
+  workflowOverview: WorkflowOverview;
+}) => {
+  const formattedWorkflowOverview = React.useMemo(
+    () => WorkflowOverviewFormatter.format(workflowOverview),
+    [workflowOverview],
+  );
+  return (
+    <div style={{ maxHeight: '300px', overflowY: 'auto' }}>
```

Comment:
Why 300px?
This will make the UI look awkward in screens with high DPI support

Reply:
replacing with rem